src/test/resources/org/broadinstitute/hellbender/tools/copynumber/gcnv-postprocess/denoised_copy_ratios_SAMPLE_000.tsv

```diff
@@ -0,0 +1,674 @@
+@HD	VN:1.6
```

Comment:
Can you add a comment in https://github.com/broadinstitute/gatk/issues/4007 to briefly describe what you did to regenerate this test data?

Reply:
Yes!